api/server/router/container/container.go

```diff
@@ -41,15 +41,18 @@ func (r *containerRouter) initRoutes() {
 		// HEAD
 		router.NewHeadRoute("/containers/{name:.*}/archive", r.headContainersArchive),
 		// GET
-		router.NewGetRoute("/containers/json", r.getContainersJSON),
+		router.NewGetRoute("/containers", r.getContainers),
+		router.NewGetRoute("/containers/json", r.getContainers),
```

Comment:
We should mark (and separate maybe ?) those with a comment to say they are deprecated and to not forgot to remove them at some point :angel: